### ISSUE TITLE
Download subtitles

### ISSUE THREAD

**Issue by forteller** [carsonip/Penguin-Subtitle-Player#23]
There's a program that lets you choose a video file and then it automatically finds srt-files for it based on file name. Then you choose which file (with which language) you want and download it/them with one click. It's super easy and great, except that it's no longer maintained and it often just throws error messages at you.

If you could take over this functonality so you both download and display srts, that would be amazing!

Thanks!

PS: the program is Free Software, maybe you can learn something from it to help you implement this? http://www.subdownloader.net/

**Comment by carsonip** [comment]
@forteller thanks for the suggestion. The Penguin Subtitle Player was originally designed for online videos so we assume that users do not have the video file. It is still not a bad idea to implement such a function tho.

**Comment by forteller** [comment]
@carsonip Thanks for the positive reply! :)

You wouldn't necessarily need to use the file name, though. You could use the <title> or other metadata from the website or just ask people to write the title of the film themselves.

**Comment by alexhk90** [comment]
@forteller It's an interesting idea though it might be better as a browser plugin as that could then automatically read the metadata from the webpage? I guess the benefit of having Penguin Subtitle Player do it is that it could then immediately load the file instead of having to manually locate it.

**Comment by forteller** [comment]
@alexhk90 Yes, that's exactly what I thought. Just tell Penguin what you want and it does everything else for you automatically. That would be so great! 

Oh, btw: I wrote "You could use the title or other metadata" above, but since I wrote "title" as a html tag it was removed. :)

**Comment by oldk1331** [comment]
There is an open source software subliminal which can download subtitles automatically, it can download subtitles without media files, given filename is enough.

https://github.com/Diaoul/subliminal
